applications/launchpad/gui-react/src/config/wallet.ts

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
seems a bit random but there's a reason I guess!

Reply:
Since there is only one thing there, then we can export it by default and use `import WalletConfig from '../../config/wallet`.

Reply:
I was actually asking about the default values themselves but just curious.